spring-cloud-dataflow-admin-starter/src/main/java/org/springframework/cloud/dataflow/admin/AdminApplication.java

```diff
@@ -16,31 +16,21 @@
 
 package org.springframework.cloud.dataflow.admin;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-
 import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.boot.autoconfigure.security.oauth2.OAuth2AutoConfiguration;
+import org.springframework.context.annotation.Configuration;
 
 /**
  * Bootstrap class for Spring Cloud Data Flow Admin.
  *
  * @author Mark Fisher
+ * @author Josh Long
  */
-@SpringBootApplication
-@EnableAutoConfiguration(exclude = {
-		OAuth2AutoConfiguration.class
-})
+@Configuration
+@EnableDataflowAdminServer
 public class AdminApplication {
 
 	public static void main(String[] args) {
-		List<String> argsList = new ArrayList<>();
-		argsList.addAll(Arrays.asList(args));
-		argsList.add("--spring.cloud.bootstrap.name=admin");
```

Comment:
I think we still require to set this property to set the config yml pointing to `admin.yml`, Isn't this no longer needed?